screencap.gemspec

```diff
@@ -17,7 +17,6 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency 'rspec', '~> 2.10'
   gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'phantomjs.rb'
```

Comment:
Why remove this?

Reply:
phantomjs.rb and phantomjs conflict with each other as they define the same classes, but have different implementations. This meant development and production version of the gem would be running different code.

The upstream screencap repository does not have phantomjs.rb included in the gemspec, which was removed from master for the same reason.